Dockerfile.j2

```diff
@@ -12,7 +12,7 @@ RUN yum install -y yum-utils deltarpm \
     && yum makecache \
     && yum install -y python3 openssl \
     && yum upgrade -y \
-    && pip3 install "pip>=19,<19.3" --upgrade \
+    && pip3 install "pip<21.2" --upgrade \
```

Comment:
Why do we need to pin the version at all?

Reply:
Thank you, I've removed the version pinning. Now, the process will select `pip 21.3.1`. I've exercised a few `pip install` commands with it and it looks like it works without issues.